launch/dynamo-run/src/main.rs

```diff
@@ -45,7 +45,11 @@ fn main() -> anyhow::Result<()> {
         },
         Err(_) => "info", 
     };
-    std::env::set_var("DYN_LOG", log_level);
+
+    if log_level != "info" {
+        std::env::set_var("DYN_LOG", log_level);
+    }
```

Comment:
You should only set the environment variable if `-v` or `-vv` are provided. In most cases you shouldn't change it. `DYN_LOG` can take much more than a single log level, see here (where it's called `RUST_LOG`) for examples: https://docs.rs/env_logger/latest/env_logger/#enabling-logging

Reply:
Thanks for mentioning this. As per the current code, it will set the `DYN_LOG` env value only if `-v` or `-vv` is provided.

Reply:
I see what you mean now. All good thanks.